### DISCUSSION TITLE
[BMP Support] Add support for BMP files.

### DISCUSSION THREAD

**Discussion by Arlind-dev** [immich-app/immich#5194] [Feature Request]
### BMP Support

I was in the process of importing older images, and for some reason, we have around 200 BMP images. Unfortunately, it appears that Immich does not support this format. I would like to keep the original files without converting them to JPEGs or PNGs, which is why I am submitting this Feature Request.

[Here](https://cdn.discordapp.com/attachments/813433779351584778/1176228659271573565/sc0001.bmp) a sample if you need one (this is the only BMP image I have where my father isn't present).

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by jrasm91** [comment]
Implemented in #5197.
